dali/operators/reader/file_reader_op.cc

```diff
@@ -29,7 +30,8 @@ This operator can be used in the following modes:
 
 In this mode, the directory indicated in ``file_root`` argument should contain one or more
 subdirectories. The files in these subdirectories are listed and assigned labels based on
-lexicographical order of the subdirectory.
+lexicographical order of the subdirectory. If you provide ``file_filters`` argument with
+list of glob strings, the operator will list files matching at least one of the patterns.
```

Comment:
```suggestion
a list of glob strings, the operator will list files matching at least one of the patterns. Otherwise, a default set of filters is used (see the default value of ``file_filters`` for details).
```
I'd also mention what happens by default